rollup.config.js

```diff
@@ -12,7 +13,11 @@ const config = {
 if (env === 'es' || env === 'cjs') {
   config.output = { format: env, indent: false }
   config.external = ['symbol-observable']
+  config.paths = env === 'es' ? {
+    'symbol-observable': 'https://unpkg.com/symbol-observable?module'
+  } : config.paths
```

Comment:
`config.paths` is always undefined here.

Reply:
For commonJS build yes, it is, but it was it before so it is not doing anything new. Anyway, might be clearer to have something like 

```js
if (env === 'es') config.paths = { 'symbol-observable': 'https://unpkg.com/symbol-observable?module' }
```

Reply:
@YerkoPalma Can we use that form? It'll be easier to understand for most folks.

Reply:
No problem 👍